### PR TITLE
Update BMLacq306.xml

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Updates.

### CODE REVIEW

File: FlorenceBML/BMLacq306/BMLacq306.xml [hunk 1]
```diff
@@ -838,6 +851,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
             <change when="2023-02-28" who="CH">updated msParts</change>
             <change when="2023-03-01" who="CH">updated msParts</change>
             <change when="2023-03-02" who="CH">updated msParts and additions</change>
+            <change when="2023-03-15" who="CH">Identification of scribe, owner insitution</change>
```
Comment (eu-genia) [inline FlorenceBML/BMLacq306/BMLacq306.xml:854]: I guess `who = "DN"`
it probably is good to also add a `<provenance>` in `<history>` , thank you!

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: FlorenceBML/BMLacq306/BMLacq306.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply FlorenceBML/BMLacq306/BMLacq306.xml:854]: I did, didn't I (for the entire volume)?

**Review by eu-genia — APPROVED** [review]

**Review by CarstenHoffmannMarburg — CHANGES_REQUESTED** [review]
"trained" in line 512

**Review by CarstenHoffmannMarburg — COMMENTED** [review]
I am totally Ok with your changes. Alas I don't know, how to switch my review to accepted.

**Comment by eu-genia** [comment]
You cannot "accept" because the PR had been already merged

**Comment by eu-genia** [comment]
all done here )

**Pull request merged**